src/chainparams.cpp

```diff
@@ -77,6 +77,7 @@ class CMainParams : public CChainParams {
         consensus.BIP34Height = 227931;
         consensus.BIP34Hash = uint256S("0x000000000000024b89b42a942fe0d9fea3bb44ab7bd1b19115dd6a759c0808b8");
         consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+        consensus.SegWitHeight = 2000000000;
```

Comment:
"some long time in the future" (as we don't have a planned activation date for segwit on mainnet yet). actual segwit will be deployed using versionbits, so this will go away